CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Changelog
-## 4.2.0-beta.0 (Not yet published)
+## 4.2.0 (Not yet published)
```

Comment:
@emclaug2  Do we need to add date here? Since in this file for previous versions we don’t have date mentioned?

Reply:
Yep if you update this we can manually publish today.